### PR TITLE
Given 2 numpy arrays return unique elements

### LINKED ISSUE #52: Given 2 numpy arrays return unique elements
# ML

Compare the elements at same index in array1 and array2 , if they dont match , append the array1 element to the output array

### Example
array1 = np.array([1,2,3,4,5,6,7])
array2 = np.array([3,2,3,8,6,6,7])

the output array should be ([1,4,5]) 
explanation : at index 0 , array1 and array2 have different elements , append the element of the first array;

### PR DESCRIPTION
**Pull request by aadityavvij**
Resolved Given 2 numpy arrays return unique elements #52

### CODE REVIEW

**Comment by ken1000minus7** [comment]
@aadityavvij you should make a separate branch before starting work on an issue, as now your pr has changes for both the issues, u need to remove the files that are not related to this issue

**Review by uttammittal02 — CHANGES_REQUESTED** [review]
Try to do this using `np.unique()` without a loop

**Comment by aadityavvij** [comment]
Tried a lot sir, but I am getting the wrong answer with np.unique() if I put repeated elements in array1 
@uttammittal02

**Comment by uttammittal02** [comment]
> Tried a lot sir, but I am getting the wrong answer with np.unique() if I put repeated elements in array1 @uttammittal02

You have 2 np arrays and you can make a 2d array consisting of both of them and then use `np.unique()` on that 2d array to get the required result

**Comment by aadityavvij** [comment]
sir, but in 2d arrays np.unique() is giving repeated rows or columns and not giving repeated elements in rows or columns

**Comment by ken1000minus7** [comment]
You can remove `np.unique()` now since it would remove repeated elements of the ans array, the rest code's fine @aadityavvij

File: ml/Given 2 numpy arrays return unique elements.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+array1 = np.array([1,2,3,4,5,6,7])
+array2 = np.array([3,2,3,8,6,6,7])
+
+if(array1.size==array2.size):
+    print(np.unique(array1[np.where(array1!=array2)]))
```
Comment (uttammittal02) [inline ml/Given 2 numpy arrays return unique elements.py:7]: Seems good
Also, you can do this in one more way
`print(array1[array1 == array2])`

**Review by uttammittal02 — APPROVED** [review]
Lgtm

File: ml/Given 2 numpy arrays return unique elements.py (hunk 1 above)
Comment (uttammittal02) [inline ml/Given 2 numpy arrays return unique elements.py]: One more method you can use to do this is `print(array1[array1 != array2])`

**Pull request merged**